DataFormats/CSCDigi/src/classes_def.xml

```diff
@@ -39,7 +39,9 @@
    <class name="CSCALCTPreTriggerDigi" ClassVersion="10">
     <version ClassVersion="10" checksum="2925979693"/>
    </class>
-   <class name="CSCShowerDigi" ClassVersion="11">
+   <class name="CSCShowerDigi" ClassVersion="13">
+    <version ClassVersion="13" checksum="2873709471"/>
```

Comment:
Hi @kakwok , why do you need to increase by two the versioning?

Reply:
we made the some major changes twice on CSC shower digi data format during the internal code development and  debug.  And the classversion and checksum are taken from compile suggestions

Reply:
@tahuang1991 I think we can just use 12 and the checksum, we don't need to skip version 12. 
@clacaputo I'm preparing the update.